x-pack/metricbeat/module/mssql/connection.go

```diff
@@ -7,6 +7,9 @@ package mssql
 import (
 	"database/sql"
 
+	// Register driver.
+	_ "github.com/denisenkom/go-mssqldb"
```

Comment:
The driver was previously registered in `fetcher.go` file, which is now removed.